server/dbSchemas/attendanceSchema.js

```diff
@@ -3,12 +3,24 @@ const { Schema } = mongoose;
 let ObjectId = mongoose.Schema.Types.ObjectId;
 
 const userSchema = new Schema({
-    email: { type: String, required: true, unique: true, trim: true },
+    email: { type: String, required: true, trim: true },
     name: { type: String, required: true, trim: true },
     password: { type: String, required: true },
-    is_professor: { type: Boolean, required: true }
+    is_professor: { type: Boolean, required: true },
+    date_created: { type: Date, required: true }
 });
-const User = mongoose.model('user', userSchema, 'users');
+const User = mongoose.model('User', userSchema, 'user');
+
+const seatingLayoutSchema = new Schema({
+    name: { type: Number, required: true },
+    capacity: { type: Number, required: true },
+    dimensions: [{ type: Number, required: true }, { type: Number, required: true }],
+    layout: [[{ type: Number, required: true }]], 
+    default: { type: Boolean, required: true },
+    description: { type: String },
+    date_created: { type: Date, required: true }
+})
+const User = mongoose.model('SeatingLayout', seatingLayoutSchema, 'seatingLayout');
```

Comment:
Here's a copy-paste typo. The name of the variable shouldn't be User since that's already defined above. (That's why Travis failed)
```suggestion
const seatingLayoutSchema = mongoose.model('SeatingLayout', seatingLayoutSchema, 'seatingLayout');
```